showcase/src/app/welcome/components/news-item/_news-item.component.scss

```diff
@@ -1,5 +1,42 @@
 .section-border {
   padding-bottom: 26px;
-  border-bottom: 1px solid mat-color($grey-palette, 400);
-  margin-bottom: 20px;
+  position: relative;
+  min-height: 1px;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  width: 290px;
+  padding-left: 12px;
+  padding-right: 12px;
+}
+
+.news-image {
+  background-size: contain;
+  height: 150px;
+  width: 290px;
+  display: block !important;
+  border: 1px solid mat-color($grey-palette, 700);
```

Comment:
If you use palettes in these styles then this file should be renamed adding the `theme` prefix as we do with the others and import it in the `showcase/src/styles/_theme.scss`